### ISSUE TITLE
Popover placement is wrong when using 'container' option

### ISSUE THREAD

**Issue by RajaJaganathan** [sandywalker/webui-popover#167]
Popover placement is wrong when set 'container' options. It is working fine without container option but accessibility point of view popover content should be siblings of popover trigger element . Please help me out. Please find below snippet.

```
<style>
        body {
            width: 940px;
            margin: 0 auto;
        }
        
        .box {
            float: right;
            margin: 100px;
        }
    </style>
```

```
<div id="box1" class="box">
  <button data-tooltip="popover">
    Click me
  </button>
  <div id="table-wrapper" style="display:none;">
    <div class="new-wrapper">
      <p style="width:250px;">Lorem ipsum dolor sit amet</p>
    </div>
  </div>
</div>

```

```
$(function() {
    $('[data-tooltip="popover"').webuiPopover('destroy').webuiPopover({
        content: function() {
            return $("#table-wrapper").html();
        },
        placement: "bottom",
        container: $(".box") //NOTE HERE
    });
    $('[data-tooltip="popover"').on('click', function(e) {
        e.stopPropagation();
    });
});

```

demo : https://jsfiddle.net/tu4ttweq/8/

version : webui popover plugin  - v1.2.8

**Comment by sandywalker** [comment]
This bug is fixed in 1.2.9, Thanks!
